### PR TITLE
Auth layout images Design 1.0

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

File: app_images/.DS_Store
Comment (alleneubank) [inline app_images/.DS_Store:1]: Nice Mac bro

**Review by alleneubank — APPROVED** [review]
The images are wayyyy too big. Hope you have a plan to optimize them. Ideally they need to be less than 100kb.

File: app_images/.DS_Store
Comment (youngkidwarrior) [reply app_images/.DS_Store:1]: Ah shit 😂

**Comment by youngkidwarrior** [comment]
Looks like I committed the uncompressed ones smfh. I'll update rn

Edit: The images are about 200kb-300kb. They are a bit large, however they are progressive jpegs, so they will not slow down initial load. A blurry image will load as a placeholder. Mobile will have some pretty intense zoom on these images, so I think it has to be this size to still look really good. 

The other option could be to load different, pre-zoomed images for mobile.

**Pull request merged**